pages/contact.tsx

```diff
@@ -161,6 +161,7 @@ export default function ContactPage() {
             id="contact_type"
             name="contact_type"
             select
+            size="medium"
```

Comment:
```suggestion
```
Hi @shondriane it seems this size `medium` is not necessary, because thats the default behavior for TextFields.
Can you check if its really necessary?

Reply:
Hi @LeoFuna the size is  small for the arrow icon as a default so I have to change it to medium so it "doubles" the size for accessibility reasons.

Reply:
Got it! :heavy_check_mark: